google/cloud/forseti/services/client.py

```diff
@@ -797,3 +799,17 @@ def delete_model(self, model_name):
         """
 
         return self.model.delete_model(model_name)
+
+    @property
+    def interceptors(self):
+        """gRPC client interceptors.
+
+        Returns:
+            tuple: A tuple of interceptors.
+        """
+        interceptors = []
+        if opencensus_enabled():
```

Comment:
What happens if `enable_tracing` is disabled on the server?  Is there any impact here in the client?

Reply:
Can we just leave a comment here?
```
# It's okay for this to be enabled on the client, even if the tracing flag is disabled on the server.
```